### PR TITLE
REF: BlueUseAllFundsButton hooks

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/a69n1mvntpn8zfdvfu2rp8729w

[download apk](http://filestorage.bluewallet.io:1488/1608331318.apk)

File: BlueComponents.js [hunk 1]
```diff
@@ -962,30 +962,50 @@ export const BlueSpacing10 = props => {
...
-  static InputAccessoryViewID = 'useMaxInputAccessoryViewID';
-  static propTypes = {
+export const BlueUseAllFundsButton = ({ balance, canUseAll, onUseAllPressed }) => {
+  const { colors } = useTheme();
+  BlueUseAllFundsButton.InputAccessoryViewID = 'useMaxInputAccessoryViewID';
+  BlueUseAllFundsButton.propTypes = {
     balance: PropTypes.string.isRequired,
     canUseAll: PropTypes.bool.isRequired,
     onUseAllPressed: PropTypes.func.isRequired,
   };
```
Comment (limpbrains) [inline BlueComponents.js:972]: I don't think defining proptypes in render function is a good idea. Same for `BlueUseAllFundsButton.InputAccessoryViewID` constant.

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5fguxffvjccr6fyknx4c1rvgqc

[download apk](http://filestorage.bluewallet.io:1488/1608368595.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
conflict

**Comment by marcosrdz** [comment]
> conflict

resolved

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vvgmzr625uupu23jb4enw093wc

[download apk](http://filestorage.bluewallet.io:1488/1608413014.apk)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/f5y3g42gcz784zka8412m09yx8

[download apk](http://filestorage.bluewallet.io:1488/1608659114.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
conflicts

**Comment by marcosrdz** [comment]
> conflicts

resolved

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wpmfw291u13bz0erbf32546h9m

[download apk](http://filestorage.bluewallet.io:1488/1608917642.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
conflicts

**Comment by marcosrdz** [comment]
> conflicts

addressed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/et57f0wgmt0630gfj26mxvxqu0

[download apk](http://filestorage.bluewallet.io:1488/1609775496.apk)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**